src/streaming/controllers/BufferController.js

```diff
@@ -152,8 +153,10 @@ function BufferController(config) {
             errHandler.mediaSourceError('Error creating ' + type + ' source buffer.');
         }
 
+        quality = requiredQuality === -1 ? 0 : requiredQuality;
```

Comment:
Correction - it appears to be any stream which doesn't have an initial quality stored.